bindings/python/htcondor2/_schedd.py

```diff
@@ -294,6 +320,142 @@ def act(self,
+    ) -> classad.ClassAd:
+        """
+        Enable User record(s) to the *condor_schedd* daemon.
+
+        :param user_spec: Which user(s) to enable.  A :class:`str`
+             of the username, a :class:`list` of such
+             strings, or a :class:`classad2.ExprTree` constraint
+        :return:  A ClassAd describing the changes made.  This
+                  ClassAd is currently undocumented.
+        """
+
+        result = userrec_act_dispatcher(self._addr, user_spec,
+            _schedd_act_on_userrec_list, _schedd_act_on_userrec_constraint,
+            (541, None)
+        )
+        return result;
+
+    def disableUserRec(self,
+        user_spec : Union[List[str], str, classad.ExprTree],
+        reason : str = None
+    ) -> classad.ClassAd:
+        """
+        Disable User record(s) in the *condor_schedd* daemon.
+
+        :param user_spec: Which user(s) to enable.  A :class:`str`
+             of the username, a :class:`list` of such
+             strings, or a :class:`classad2.ExprTree` constraint
+        :param reason: A free-form justification.  Defaults to
+            "Python-initiated action"
+        :return:  A ClassAd describing the changes made.  This
+                  ClassAd is currently undocumented.
+        """
+
+        if reason is None:
+            reason = "Python-initiated action"
+
+        result = userrec_act_dispatcher(self._addr, user_spec,
+            _schedd_act_on_userrec_list, _schedd_act_on_userrec_constraint,
+            (542, reason)
+        )
+        return result;
+
+    def removeUserRec(self,
+        user_spec : Union[List[str], str, classad.ExprTree],
+        reason : str = None
+    ) -> classad.ClassAd:
+        """
+        Remove User record(s) in the *condor_schedd* daemon.
+
+        :param user_spec: Which user(s) to enable.  A :class:`str`
+             of the username, a :class:`list` of such
+             strings, or a :class:`classad2.ExprTree` constraint
+        :param reason: A free-form justification that is used when
+             the record cannot be removed.  Defaults to
+            "Python-initiated action"
+        :return:  A ClassAd describing the changes made.  This
+                  ClassAd is currently undocumented.
+        """
+
+        if reason is None:
+            reason = "Python-initiated action"
+
+        result = userrec_act_dispatcher(self._addr, user_spec,
+            _schedd_act_on_userrec_list, _schedd_act_on_userrec_constraint,
+            (549, reason)
+        )
+        return result;
+
+
+    def addProjectRec(self,
+        project_spec : Union[List[str], str, List[classad.ClassAd] ],
+    ) -> classad.ClassAd:
+        """
+        Add Project record(s) to the *condor_schedd* daemon.
+
+        :param project_spec: Which projects(s) to add.  A :class:`str`
+             of the project name, a :class:`list` of such
+             strings, a :class:`classad2.ClassAd` of project ads
```

Comment:
To match the documentation for `addUserRec()`:
```suggestion
        :param project_spec: Which projects(s) to add.  A :class:`str`
             of the project name, a :class:`list` of such
             strings, or a :class:`list` of initial project records in class:`classad2.ClassAd` form.
```